### ISSUE TITLE
Service - API REST call with GET params

### ISSUE THREAD

**Issue by AlbertFX91** [AlbertFX91/Deccom#63]

**Comment by juagarfer4** [comment]
This was a pure mess lol.

First of all, I could not use the books mapping that was implemented before, as I could not add any parameters to the URL when sending requests to thart particular API. Therefore, I had to map the posts from JSONPlaceholder (yes, could have opened the previous issue, but...), and adding the proper parameter to the request in order to get the posts from a specific user.

The results is quite similar, but the point is that we have been able to add a parameter to the URL and obtain satisfying results. Of course, depending on the parameters and, first and foremos, on the API, the mapping would be different. That would be different for each API and maybe each request (not so much in this second case).

[Code](https://gist.github.com/juagarfer4/b1a03586807dc112c5b81f3571853847)
[Proof](https://gyazo.com/07ac65c3ba8b219cb1a5230bea906fa7)
